broker-cli/proto/broker.proto

```diff
@@ -1787,6 +1787,20 @@ message ChangeWalletPasswordRequest {
   string new_password = 3;
 }
 
+/**
+ * Request to recover a wallet
+ */
+message RecoverWalletRequest {
+  // Currency of wallet
+  string symbol = 1;
+  // Password from the existing wallet
+  string password = 2;
+  // 24 word mnemonic seed
+  repeated string seed = 3;
+  // The path of the backup file
+  bool backup = 4;
```

Comment:
`use_backup` might be better. Also, maybe something to make it more generic, like `recover_committed`?

Reply:
I agree the name can change. I like `recover-committed`, but going to think on it for a bit